### DISCUSSION TITLE
Headbar flickering on safari iOS

### DISCUSSION THREAD

**Discussion by aliasion** [GitbookIO/community#290] [0. General bugs and questions] [answered]
The headbar is flickering on iOS Safari or Chrome (On Android is working fine)

See the attached video (Note that the flickering is also happening on the pages generated by me)

https://github.com/GitbookIO/community/assets/36562834/8b01d2ef-ccff-47b3-9ad9-7d9b047a8e5a

**Comment by aikoii** [comment, marked as answer]
Hey @atelog! 👋 I'm testing it out on iOS 16.6 Safari and Chrome, and it looks like the problem has been resolved (we made some changes over the weekend). Does it work for you without any flickering now?

**Reply by aliasion** [reply]
I've tested today (from Incognito) and the issue was still present. I've upgraded the iOS from 16.1 to 16.6 and now there is no more flickering.
